rotkehlchen/globaldb/schema.py

```diff
@@ -249,15 +249,15 @@
 {DB_CREATE_ASSET_TYPES}
 {DB_CREATE_ASSETS}
 {DB_CREATE_EVM_TOKENS}
-{DB_CREATE_MULTIASSETS}
+{DB_CREATE_MULTIASSET_MAPPINGS}
 {DB_CREATE_COMMON_ASSET_DETAILS}
 {DB_CREATE_USER_OWNED_ASSETS}
 {DB_CREATE_PRICE_HISTORY_SOURCE_TYPES}
 {DB_CREATE_PRICE_HISTORY}
 {DB_CREATE_BINANCE_PAIRS}
 {DB_CREATE_ADDRESS_BOOK}
 {DB_CREATE_CUSTOM_ASSET}
-{DB_CREATE_ASSET_COLLECTION_PROPERTIES}
+{DB_CREATE_ASSET_COLLECTIONS}
```

Comment:
Shouldn't this table be created before `DB_CREATE_MULTIASSET_MAPPINGS`?

Reply:
Doesn't make sense since foreign keys are off anyway and other tables are also created in a random order